group.hpp

```diff
@@ -77,8 +77,15 @@ class Group : public GroupInherit
     Serialize& serialize;
 
     /** @brief Custom callback when LED group is asserted
+     * Callback that holds LED group method with below mentioned parameter and
+     * return type.
+     *
+     * @param[in] group object - Pointer to Group object
+     * @param[in] boolean - Input value (true/false)
+
+     * @return Call back execution success(true) or failure(false).
```

Comment:
* @return  bool

Reply:
done. it should be fine if I explain the return statement

Reply:
yes , just that you need to mention boolean return , no need for success true and failure false